types/src/proto/transaction.proto

```diff
@@ -73,16 +43,8 @@ message Module {
 
 // A generic structure that represents signed RawTransaction
 message SignedTransaction {
-    // The serialized Protobuf bytes for RawTransaction, for which the signature
-    // was signed. Protobuf doesn't guarantee the serialized bytes is canonical
-    // across different language implementations, but for our use cases for
-    // transaction it is not necessary because the client is the only one to
-    // produce this bytes, which is then persisted in storage.
-    bytes raw_txn_bytes = 1;
-    // public key that corresponds to RawTransaction::sender_account
-    bytes sender_public_key = 2;
-    // signature for the hash
-    bytes sender_signature = 3;
+    // LCS byte code representation of a SignedTransaction
+    bytes signed_txn = 5;
```

Comment:
5?

Reply:
we should reset this to 0 at some point in time.... 4 was used in an early commit in this stack.